addon/components/uk-toggle-switch.js

```diff
@@ -0,0 +1,57 @@
+import Component from "@ember/component";
+import layout from "../templates/components/uk-toggle-switcher";
+import validatedComputedProperty from "ember-uikit/-private/validated-computed-property";
+import { computed } from "@ember/object";
+
+export const SIZE_OPTIONS = {
+  SMALL: "small",
+  MEDIUM: "medium",
+  LARGE: "large"
```

Comment:
Do you think we can start considering ES6 symbols for cases like this? (See "Okay, but what are Symbols really good for?" on https://www.keithcirkel.co.uk/metaprogramming-in-es6-symbols/)
They are polyfilled by babel-polyfill, so browser compat might not be an issue.

Reply:
Interesting idea! However, I don't really see how we can use it here. The values in there need to be passed to the DOM element as a `String`. As far as I understood `Symbols`, they do not have a primitive value.. Or did I miss something?

Reply:
Ah, you're right - I didn't look closely enough. `Symbols` are handy when you're not interested in the actual value, but only want the values to be unique; for example when you're trying to emulate Enums.